blog-service/2022-11-09-traces.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Tracing - Environment Support for APM Dashboards
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - sumo logic
+  - service release notes
+  - traces
+  - apm
+hide_table_of_contents: false
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Update - Tracing dashboards now offer native support for monitoring OpenTelemetry using the `deployment.environment` standard tag.
```

Comment:
native support for OpenTelemetry's

Reply:
Is this right @pbrzoska-sumo ?

Update - Tracing dashboards now offer native support for OpenTelemetry's `deployment.environment` standard tag.